assets/_scss/themes/_usa.scss

```diff
@@ -16,13 +17,13 @@ li {
 }
 
 a {
-  color: $color-main;
+  color: $color-primary;
```

Comment:
Someone would just add another theme file to the themes folder to change their custom theme. Also variables does not contain any live code or styling, just variables. The theme files contain the implementation of variables or other customizations.